### PR TITLE
Auth login

### PR DESCRIPTION
**Pull request by KatrinDav**

### CODE REVIEW

File: src/components/FormLogin.vue [hunk 1]
```diff
@@ -66,13 +73,19 @@ export default {
     },
   },
   methods: {
-    onSubmit() {
+    async onSubmit() {
       if (this.isFormValid) {
         console.log(this.email, this.password);
+        this.$store.dispatch("login", {
```
Comment (rossinek) [inline src/components/FormLogin.vue:79]: it may be nice to somehow handle possible errors here

File: src/helpers/auth.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { defaultClient } from "./client";
+
+export const login = async (credentials) =>
```
Comment (rossinek) [inline src/helpers/auth.js:3]: I really like the fact of separating this file and the `client.js` file, but I'm not sure if the `helpers` directory name is well suited 🤔

File: src/store/index.js [hunk 3]
```diff
@@ -1,20 +1,33 @@
...
+
+const AUTH_HEADER_STORAGE_KEY = 'auth-header';
 
 Vue.use(Vuex)
 
 export default new Vuex.Store({
   state: {
     movies: [],
     screenings: [],
+    authHeader: null,
```
Comment (rossinek) [inline src/store/index.js:15]: In the future I would consider creating a separate `user` module

**Review by rossinek — APPROVED** [review]
Looks very good! 👏  approved with small tips

File: src/helpers/auth.js (hunk 2 above)
Comment (KatrinDav) [reply src/helpers/auth.js:3]: Ok. I'll make "api" directory for these file.

File: src/components/FormLogin.vue (hunk 1 above)
Comment (KatrinDav) [reply src/components/FormLogin.vue:79]: Error handling will be added.

File: src/store/index.js (hunk 3 above)
Comment (rossinek) [reply src/store/index.js:15]: https://vuex.vuejs.org/guide/modules.html
https://vuex.vuejs.org/guide/modules.html#namespacing

**Pull request merged**